src/main/java/uk/gov/pay/adminusers/service/NotificationService.java

```diff
@@ -61,11 +61,18 @@ NotifyDirectDebitConfiguration getNotifyDirectDebitConfiguration() {
 
     CompletableFuture<String> sendSecondFactorPasscodeSms(String phoneNumber, String passcode) {
         return CompletableFuture.supplyAsync(() -> {
+            // convert back to local format for backward compatibility
+            String formattedPhoneNumber;
+            if (phoneNumber.startsWith("+44")) {
+                formattedPhoneNumber = phoneNumber.replaceFirst("\\+44", "0");
+            } else {
+                formattedPhoneNumber = phoneNumber;
+            }
```

Comment:
Is this change going to be reverted after the end-to-end tests have been updated?

Reply:
Yes, it will be reverted once we migrate fully, thank you for asking